src/Pororoca.Domain/Features/VariableCapture/PororocaResponseValueCapturer.cs

```diff
@@ -7,9 +7,12 @@ namespace Pororoca.Domain.Features.VariableCapture;
 public static partial class PororocaResponseValueCapturer
 {
     private static readonly Regex arrayElementRegex = GenerateArrayElementRegex();
+    private static readonly Regex firstWhereRegex = FirstWhereRegex();
 
     [GeneratedRegex("\\[(\\d+)\\]")]
     private static partial Regex GenerateArrayElementRegex();
+    [GeneratedRegex("firstWhere\\((.+)\\)")]
```

Comment:
Regex is a good idea for this case.

Tips:
- We can use [named captures](https://learn.microsoft.com/en-us/dotnet/standard/base-types/grouping-constructs-in-regular-expressions#named-matched-subexpressions) so the regex delivers the property name and expected value. Like: `(?<PropertyName>subexpression)`
- PropertyName can only have digits, chars (a-z, A-Z) and underscores (_)
- Whitespaces are accepted inside the expression: `firstWhere( id == 1 )`